StudentProject/StudentDetails.java

```diff
@@ -48,88 +27,86 @@ public static void main(String[] args) {
             ch = s.nextInt();
 
             switch (ch) {
-                case 1:
+                case ADD_STUDENT:
                     System.out.println("--------------------------------------------------");
                     System.out.println("Please provide the data for the student to store. ");
                     System.out.print("Student id: ");
-                    int sno = s.nextInt();
+                    int eno = s.nextInt();
                     System.out.print("First name: ");
-                    String sfname = s1.nextLine();
+                    String StudentFirstName = s1.nextLine();
```

Comment:
The Java convention for variables is lowerCamelCase (https://www.techtarget.com/whatis/definition/lowerCamelCase).
Use studentFirstName instead of StudentFirstName ...